src/main/java/by/onliner/utils/Waiter.java

```diff
@@ -22,4 +25,14 @@ public static void elementToBeClickable(WebElement element) {
     public static void waitForVisibility(WebElement element) {
         wait.until(ExpectedConditions.visibilityOf(element));
     }
+
+    /***
+     * Waits until the element is displayed using custom timing
+     */
+    public static void waitForVisibility(WebElement element, int timeoutSec, int pollingMils) {
```

Comment:
Create a method where you pass only timeout value as an argument and polling is default. The chance that you will need custom polling value is very law.

Reply:
Ok